protocol/synthetix/subgraph/subgraph.base-mainnet-andromeda.yaml

```diff
@@ -52,10 +52,6 @@ dataSources:
           handler: handlePoolConfigurationSet
         - event: MarketRegistered(indexed address,indexed uint128,indexed address)
           handler: handleMarketCreated
-        - event: MarketUsdDeposited(indexed uint128,indexed address,uint256,indexed address,int128,int128,uint256,uint256)
-          handler: handleMarketUsdDeposited
-        - event: MarketUsdWithdrawn(indexed uint128,indexed address,uint256,indexed address,int128,int128,uint256,uint256)
-          handler: handleMarketUsdWithdrawn
```

Comment:
Was this removed on purpose? Should it be removed from here too https://github.com/Synthetixio/synthetix-v3/blob/fa085945b21b2b1a23ca953d45b28d5389112e95/protocol/synthetix/subgraph/base-mainnet-andromeda/index.ts#L10C27-L10C51

Reply:
Yes, these events weren't part of the CoreProxy